.eslintrc.js

```diff
@@ -115,6 +115,50 @@ module.exports = {
         ],
       },
     },
+    {
+      files: ['tests/e2e/**/*'],
+      parserOptions: {
+        project: 'tests/e2e/tsconfig.json',
+      },
+      rules: {
+        'valid-jsdoc': 'off',
+        '@typescript-eslint/no-unused-vars': [
+          'error',
+          {
+            argsIgnorePattern: '^_',
+            varsIgnorePattern: 'WebdriverIOAsync',
+            ignoreRestSiblings: true,
+          },
+        ],
+        '@typescript-eslint/no-floating-promises': 'error',
+      },
+    },
+    {
+      files: [
+        'tests/e2e/specs/**/*.spec.ts',
+        'tests/e2e/all-flavors.spec.ts',
+        'tests/e2e/helpers/**/*.ts',
+        'tests/e2e/wdio.*.conf.js',
+      ],
+      extends: ['plugin:wdio/recommended'],
+      plugins: ['wdio'],
+      env: {
+        jasmine: true,
+      },
+      rules: {
+        // we are exporting test "factories", so we need to allow export
+        'jest/no-export': 0,
+        // we use jasmine
```

Comment:
Is it required to use Jasmine with WebdriverIO? Could we do another PR to use Jest instead and get rid of it?

Reply:
Jasmine is the closest to jest's expect, but you can't use jest

Reply:
It looks like it's possible but there's no straightforward path: https://github.com/webdriverio/webdriverio/issues/2052

Anyway, let's consider it some other time.